### PR TITLE
Set deployment target to iOS 9

### PR DESCRIPTION
**Pull request by danielpassos**
Related issues: 

* [FH-3921](https://issues.jboss.org/browse/FH-3921) - Set deployment target to iOS 9 on saml-ios-app
* [FH-3952](https://issues.jboss.org/browse/FH-3952) - Move saml-ios-app from Travis CI to CircleCI

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,8 +11,8 @@ notifications:
   irc: "irc.freenode.org#feedhenry"
 
 script:
-  - xcodebuild clean build -workspace saml-ios-app.xcworkspace -scheme saml-ios-app -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 6' GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES
-
+  - xcodebuild clean build -workspace saml-ios-app.xcworkspace -scheme saml-ios-app -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 7,OS=11.0' GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES
 branches:
```
Comment (jcesarmobile) [inline .travis.yml]: I think we should remove the "only" instead of adding new branches

File: .travis.yml [hunk 2]
```diff
@@ -11,8 +11,8 @@ notifications:
   irc: "irc.freenode.org#feedhenry"
```
Comment (jcesarmobile) [inline .travis.yml]: Add 

```
before_install:
  - travis_wait pod repo update
```
That should fix the build fail.

File: README.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ Community Project : [Feed Henry](http://feedhenry.org)
 Target Product: RHMAP  
 Product Versions: RHMAP 3.7.0+   
 Source: https://github.com/feedhenry-templates/saml-ios-app  
-Prerequisites: fh-ios-sdk : 3.+, Xcode : 8+, iOS SDK : iOS8+, CocoaPods: 1.0.1+
+Prerequisites: fh-ios-sdk: 3.+, Xcode: 9+, iOS SDK: iOS 9+, CocoaPods: 1.3.0+
```
Comment (jcesarmobile) [inline README.md]: fh-ios-sdk: 4.+

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
Added a few comments
The Podfile has to be updated to iOS 9 too.

File: .travis.yml (hunk 1 above)
Comment (danielpassos) [reply .travis.yml]: Only is to prevent be annoyed when new release branch is created :/

File: .travis.yml (hunk 2 above)
Comment (danielpassos) [reply .travis.yml]: :smirk: it's already there

Comment (jcesarmobile) [reply .travis.yml]: Ah, true, then I don't know why it fails to find the dependency

**Comment by danielpassos** [comment]
@jcesarmobile Suggestions applied and added @CircleCI

**Review by jcesarmobile — APPROVED** [review]

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
As we have removed travis and added circle-ci we should change the badge.

I think this should work:

`[CircleCI](https://circleci.com/gh/feedhenry-templates/saml-ios-app)`

**Review by jcesarmobile — APPROVED** [review]

**Comment by jcesarmobile** [comment]
We can fix the badge and add some more badges latter, so merging this

**Pull request merged**